dw-hardened-ami/generic_packer_template.json.j2

```diff
@@ -6,14 +6,17 @@
     },
     "builders": [{
       "type": "amazon-ebs",
+      {% if 'source_ami_id' in event and event['source_ami_id']|length > 0 %}
+        "source_ami": "{{ event['source_ami_id']}}"",
+      {% endif %}
       "source_ami_filter": {
         "filters": {
-        "virtualization-type": "hvm",
-        "name": "amzn-ami-hvm-*",
-        "root-device-type": "ebs",
+        "virtualization-type": "{{ event['source_ami_virt_type'] or 'hvm' }}",
+        "name": "{{ event['source_ami_name'] or 'amzn-ami-hvm-*' }}",
+        "root-device-type": "{{ event['source_ami_root_device_type'] or 'ebs' }}",
         "architecture": "x86_64"
```

Comment:
Given that we are pinning all versions to AL1, hvm ebs on 64bit architecture, there is no need for the event variables anymore here. The base AL1 has to be compatible with EMR, so deviating from hvm, x86_64 or ebs will break that compatibility. That makes these event vars moot